src/ILVerify/tests/TestDataLoader.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Internal.IL;
+using Internal.TypeSystem.Ecma;
+
+namespace ILVerify.Tests
+{
+    /// <summary>
+    /// Helper method to load the binaries generated based on the il code, which drive the tests
+    /// </summary>
+    static class TestDataLoader
+    {
+        /// <summary>
+        /// The folder which contains the core assembly (containing System.Object, etc.)
+        /// </summary>
+        static string COREASSEMBLYPATH = @"..\..\..\..\..\..\bin\Product\Windows_NT.x64.Debug\CoreTestAssembly\\";
+
+        /// <summary>
+        /// The folter with the binaries which are compiled from the test driver IL Code
+        /// </summary>
+        static string TESTASSEMBLYPATH = @"..\..\..\ILTests\";
```

Comment:
You should be able to get the test files dropped to the output by referencing the Project:

https://blogs.msdn.microsoft.com/kirillosenkov/2015/04/04/how-to-have-a-project-reference-without-referencing-the-actual-binary/